README.md

```diff
@@ -109,6 +109,9 @@ Beside these options handled by Prawn / prawn-table, the following values may be
   - `:radio`
     - `:checked`: The char to print for a checked radio. Default is '◉'.
     - `:unchecked`: The char to print for an unchecked radio. Default is '○'.
+- `:link`
+  - `:color`: The link color, which can be specified in either RGB or CMYK format.
```

Comment:
As a prawn plugin, we should use the same format for this option as prawn does and document it here: "... in RGB hex format or 4-value CMYK".